tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -927,6 +935,37 @@ def push_resource(self, filename, remote_path=None, count=1):
     def resource(self, filename):
         return os.path.abspath(os.path.join(os.path.dirname(__file__), 'resources', filename))
 
+    def modify_settings(self, settings):
```

Comment:
I'm not sure `override_default_settings` is an improvement. I've added a docstring for this method, which hopefully makes it less disturbing. Unless I hear a convincing argument for an alternate method name I'll leave it as is.